tools/validators/instance_validator/validate/telemetry_validator.py

```diff
@@ -22,12 +22,20 @@
 
 from validate import telemetry
 from validate import telemetry_error
+from validate import telemetry_warning
 
 DEVICE_ID = 'deviceId'
 
 class TelemetryValidator(object):
   """Validates telemetry messages against a building config file.
 
+  Attributes;
+    entities: a dict with entity_name as a key and EntityIntance as value.
```

Comment:
entity_name doesn't mean anything to the reader here.

You could say the key is the entity name as a string.

Reply:
nit: typo for EntityIn_s_tance

Reply:
I kept entity name for now as we defined it here for the user: https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md